ChatRPG/Pages/UserCampaignOverview.razor

```diff
@@ -1,63 +1,75 @@
-                            <div class="card-body">
-                                <h5 class="card-title">@scenario.Title</h5>
-                                <p class="card-text">@scenario.Body</p>
-                                <a href="#" class="card-link">See more ></a>
-                            </div>
-                        </div>
-                    }
-                    <p class="text-muted">Found @(StartScenarios.Count) scenario@(StartScenarios.Count == 1 ? "" : "s").</p>
+                    <h3 class="text-center mb-4">Starting Scenarios</h3>
+                    <div class="vstack overflow-auto form-text-black mb-1 scrollbar" style="max-height: 400px;" id="start-scenarios">
+                        @foreach (StartScenario scenario in StartScenarios)
+                        {
+                            <Scenario ApplyScenario="ApplyStartingScenario" Title="@scenario.Title" Body="@scenario.Body"/>
+                        }
+                    </div>
+                    <p class="text-muted" id="scenarios-count">Found @(StartScenarios.Count) scenario@(StartScenarios.Count == 1 ? "" : "s").</p>
                 </div>
             </div>
             <div class="col-5 rounded custom-gray-container">
                 <div class="m-3">
-                <h3>Create a Custom Campaign</h3>
+                    <h3 class="text-center mb-4">Create a Custom Campaign</h3>
                     <div class="form">
                         <div class="mb-3">
-                            <label for="inputCharacterName" class="form-label">Character Name*</label>
-                            <input type="text" @bind="CharacterName" class="form-control" id="inputCharacterName"/>
+                            <label for="inputCampaignTitle" class="form-label">Campaign Title<span class="required-keyword">(Required)</span></label>
+                            <input type="text" @bind="CampaignTitle" @oninput="UpdateCampaignTitleOnKeyPress" class="form-control" id="inputCampaignTitle" data-toggle="tooltip" data-placement="top" title="Campaign title is required.">
+                            @if (string.IsNullOrWhiteSpace(CampaignTitle) && TestFields)
+                            {
+                                <div class="alert-sm alert-danger alert-inside-input mt-1 p-2 rounded" role="alert" id="campaign-title-alert">
+                                    Campaign title is required.
+                                </div>
+                            }
                         </div>
                         <div class="mb-3">
-                            <label for="inputCampaignTitle" class="form-label">Campaign Title*</label>
-                            <input type="text" @bind="CampaignTitle" class="form-control" id="inputCampaignTitle">
+                            <label for="inputCharacterName" class="form-label">Character Name<span class="required-keyword">(Required)</span></label>
```

Comment:
Når nu du er så godt i gang, vil du så ikke tilføje en "inputCharacterDescription" form? Så har vi alt vi skal bruge for at lave spillerens `Character`.

Reply:
Hahaha, oh nooooooo.
Det er ikke i designet!
Jeg troede faktisk at character description ville udledes af scenario, men jo, jeg kan godt tilføje et sted at skrive det. Det ville bare have været fedt at vide inden alt det jeg har lavet idag haha

Reply:
Hehe oops! Men jeg synes da at brugeren selv skal have mulighed for at beskrive deres egen character. Ellers er det ikke en rigtig RPG!

Reply:
*Sigh* I wholehearted agree, men fuck noget bøvl! Cease the pull requestos!

Reply:
Kan du ikke "bare" duplikere "CharacterName" feltet og bind det til CharacterDescription i stedet? Eller lægger der mere bag det?

Reply:
UI mæssigt kræver det lige lidt pilfingerhans, if you know what I mean 👀

Reply:
No worries. Jeg resolver

Reply:
Jeg fixer det, don't worry, min ven! Jeg har lavet et kort på kanban så jeg ikke glemmer det.

Reply:
Det var også originalt i designet, men så sagde dam at jeg skulle fjerne det. En uge senere sagde laver han en 360 og siger vi skulle have det med igen, men så glemte jeg at få opdateret designet 💀

Reply:
@damniko jeg har tilføjet CharacterDescription nu. Det var super let, så jeg ved ikke hvorfor jeg huskede det som problematisk. Det hele scaler jo ret godt fordi jeg er en UI king :karmasmug: